addons/stock/models/stock_orderpoint.py

```diff
@@ -289,6 +289,7 @@ def _set_default_route_id(self):
         orderpoints generated when openning the replenish report.
         """
         self = self.filtered(lambda o: not o.route_id)
+        archived_route_ids = self.env['stock.route'].search([('active', '=', False)]).ids
```

Comment:
I would add `(route_id.active, !=, False)` in the read_group domain on stock.rule instead to avoid extra sql request

Reply:
@Whenrow , I have applied the changes suggested by you.